dev/devicelab/manifest.yaml

```diff
@@ -795,3 +795,16 @@ tasks:
   #   stage: devicelab
   #   required_agent_capabilities: ["linux/android"]
   #   flaky: true
+
+  new_gallery_chrome_run_test:
```

Comment:
nit: mark both of these as flaky for now. After a few passing runs we can unmark them

Reply:
Done.

Reply:
Done.